src/ERC20Extended.sol

```diff
@@ -111,7 +84,7 @@ abstract contract ERC20Extended is IERC20Extended, IERC5267, ERC3009 {
         uint256 spenderAllowance_ = allowance[sender_][msg.sender]; // Cache `spenderAllowance_` to stack.
 
         if (spenderAllowance_ != type(uint256).max) {
-            if (spenderAllowance_ < amount_) revert ERC20InsufficientAllowance(msg.sender, spenderAllowance_, amount_);
+            if (spenderAllowance_ < amount_) revert InsufficientAllowance(msg.sender, spenderAllowance_, amount_);
```

Comment:
why to change error?

Reply:
We never prefix events and errors with the contract name.